ManatiCR/retopais
# RP-24: Translations

https://manati.atlassian.net/browse/RP-24

How-to-test:
- Install site
- Run `drush l10n-update-refresh`
- Run `drush l10n-update`
- Go to /contacto; everything should be in spanish
